akka-cluster-sharding-typed/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingRememberEntitiesPerfSpec.scala

```diff
@@ -177,7 +177,7 @@ abstract class ClusterShardingRememberEntitiesPerfSpec
       enterBarrier(s"after-start-stop-${testRun}")
     }
 
-    "test when starting new entity" in {
+    "test when starting new entity" taggedAs GHExcludeTest in {
```

Comment:
should we make a plan for enabling those tests again?

Reply:
Do you want me to open an issue and link it?

Reply:
Linked an issue to all of them